src/select/index.tsx

```diff
@@ -53,7 +51,9 @@ export interface SelectProperties {
 
 export interface SelectChildren {
 	/** Custom renderer for item contents */
-	(properties: ItemRendererProperties): RenderResult;
+	itemRenderer?(properties: ItemRendererProperties): RenderResult;
```

Comment:
could you rename this to `items` please? I believe this is more in line with what we've named things elsewhere, ie. `radios` / `checkboxes` etc for those rendered in the groups.